concordium-consensus/src/Concordium/GlobalState/Persistent/Bakers.hs

```diff
@@ -497,6 +518,24 @@ addDelegator (DelegateToBaker bid) did amt pab =
             newActiveBakers <- Trie.insert bid pad' (pab ^. activeBakers)
             return $ Right $ pab & activeBakers .~ newActiveBakers
 
+-- | Add a delegator to the persistent active bakers at a particular target.
+--  It is assumed that the delegator is not already delegated to this target.
+--  If the target is a baker, then the baker MUST be in the active bakers.
+--
+--  IMPORTANT: This does not update the total active capital!
```

Comment:
Because when a delegator or baker is added, the total active stake increases. One should therefore be aware that the total active stake is not updated by this function.